src/main/scala/util/HellaQueue.scala

```diff
@@ -2,35 +2,36 @@
 
 package freechips.rocketchip.util
 
-import Chisel._
+import chisel3._
+import chisel3.util._
 
 class HellaFlowQueue[T <: Data](val entries: Int)(data: => T) extends Module {
   val io = new QueueIO(data, entries)
```

Comment:
Need a `IO()` wrapper here.

The `io.count` is not connected.

Reply:
Pending fix submitted in https://github.com/chipsalliance/rocket-chip/pull/3157